docs/modules/install/pages/index.adoc

```diff
@@ -0,0 +1,63 @@
+= Getting started with Decidim Bulletin Board
+:source-highlighter: highlightjs
+
+== What is and what isn't Decidim Bulletin Board?
+
+Decidim Bulletin Board is an independent application designed to provide services to Decidim instances to allow them to celebrate secure end-to-end auditable votings. But it can't be used as a standalone application, since it doesn't offer a user interface organization administrators to define elections nor for voters to cast their votes.
```

Comment:
```suggestion
Decidim Bulletin Board is an independent application designed to provide services to Decidim instances to allow them to hold secure end-to-end auditable votings. But it can't be used as a standalone application, since it doesn't offer a user interface organization administrators to define elections nor for voters to cast their votes.
```
Ditto